sensors/triggers/params.go

```diff
@@ -274,10 +293,15 @@ func getValueWithTemplate(value []byte, templString string) (string, error) {
 
 // getValueByKey will return the value in the raw json bytes at the provided key,
 // or an error if it does not exist.
-func getValueByKey(value []byte, key string) (string, error) {
+func getValueByKey(value []byte, key string) (string, string, error) {
 	res := gjson.GetBytes(value, key)
 	if res.Exists() {
-		return res.String(), nil
+		if res.Type.String() == "String" {
+			return res.String(), res.Type.String(), nil
+		} else if res.Type.String() == jsonType {
+			return res.Raw, res.Type.String(), nil
```

Comment:
What will happen if we always use `Raw`?

Reply:
It will brake what other function expect to get ( uni-test are failing). I wanted to save the functionality of the function and not barking anything else.